net-core/docs/kickoff/README.md

```diff
@@ -32,16 +32,26 @@ First, make sure you have the latest version of the [NetCore Bootstrap](https://
     git checkout -b development
 ```
 
-Run the script that changes all the names of the base project to the one you really want. To do this run:
-
+To change all the names of the base project to the one you really want, you need to run a script.
+To do so, follow the next steps:
```

Comment:
I'd replace this line and the latter with this one:
"In order to replace in every file the 'BaseProject' name with your project name follow the next steps:"